packages/babel-helper-builder-binary-assignment-operator-visitor/package.json

```diff
@@ -14,9 +14,11 @@
   },
   "main": "./lib/index.js",
   "dependencies": {
-    "@babel/helper-explode-assignable-expression": "workspace:^",
```

Comment:
This package had some `TODO(Babel 8)` comments regarding its public interface. It was only used in `@babel/helper-builder-binary-assignment-operator-visitor`, so I inlined it in `packages/babel-helper-builder-binary-assignment-operator-visitor/src/explode-assignable-expression.ts` and made the necessary changes (basically, removing the `allowedSingleIdent` and `file` parameters from the `explode` function).

After this PR is merged, I'll move the package to https://github.com/babel/babel-archive.